doc/git-integration.md

```diff
@@ -112,10 +112,23 @@ To create and verify these signatures we simply wrap the gpg binary with our
 own code, `ots-git-gpg-wrapper`. Git allows you to override the default GnuPG
 binary (`/usr/bin/gpg`) with your own using the `gpg.program` config option.
 Unfortunately that option doesn't let you set additional command line flags, so
-we use one more wrapper, `ots-git-gpg-wrapper.sh`. You can set all this up with the
-following:
-
-    git config --global gpg.program <path to ots-git-gpg-wrapper.sh>
+we use one more wrapper, `ots-git-gpg-wrapper.sh`. You can set all this up with
+either of the following:
+
+```bash
+# just specify ots-git-gpg-wrapper.sh and let `git` find it itself
+git config --global gpg.program ots-git-gpg-wrapper.sh
```

Comment:
By "find it itself" doesn't git just use `$PATH` as normal?

Reply:
Exactly. It is possible to only specify `ots-git-gpg-wrapper.sh` without a full path if it is in the `PATH`, which is a convenience as on different machines it can be installed in different places (e.g. `~/.local/bin` or `/usr/local/bin` or `/usr/bin/`, ...). This is a strong argument for putting it into a standard executable location.